pkg/config/schedule/schedule.go

```diff
@@ -0,0 +1,111 @@
+// Copyright 2022 Allstar Authors
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package schedule provides the ActionSchedule type for use in config and
+// when deciding whether to perform an action.
+package schedule
+
+import (
+	"strings"
+	"time"
+)
+
+type ScheduleAction string
```

Comment:
Remove `Actions  ScheduleConfigActions` and this type and code. Assume/hardcode that the schedule is for "notify" actions (issue/ping), and that log/fix will always work.

Overall I'd like to follow the YAGNI principle and make things simple for the sake of code and reduce the complexity for the user setting up the config file.